code/modules/mob/living/simple_animal/friendly/iriska.dm

```diff
@@ -0,0 +1,203 @@
+/mob/living/simple_animal/iriska
+	name = "Iriska"
+	desc = "The captain's own cat. Fat and lazy."
```

Comment:
It's the other way around. Captain is Iriska's pet

Reply:
I agree. Initially there was something like "An actual captain of the ship".